review/conftest.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+from review.models import Review
+
+
+@pytest.fixture
+def daycare_fixture():
```

Comment:
It is known that this is a fixture, it is written right above it. :) `daycare` should be enough

Reply:
changed